multi/gr716-multi/gr716-multi.c

```diff
@@ -90,6 +98,23 @@ static id_t multi_getId(msg_t *msg)
 	return id;
 }
 
+#if PSEUDODEV
+static inline int multi2pseudo(id_t id)
+{
+	switch (id) {
+		default:
+		case id_pseudoNull:
+			return pseudo_idNull;
+		case id_pseudoZero:
+			return pseudo_idZero;
+		case id_pseudoFull:
+			return pseudo_idFull;
+		case id_pseudoRandom:
+			return pseudo_idRandom;
```

Comment:
Nitpick: default should be last
```suggestion
		case id_pseudoZero:
			return pseudo_idZero;
		case id_pseudoFull:
			return pseudo_idFull;
		case id_pseudoRandom:
			return pseudo_idRandom;
		case id_pseudoNull:
		default:
			return pseudo_idNull;
```

Reply:
Rule 16.5 - (required) - A default label shall appear as either the first or the last switch label of a switch statement.